vscode-react/src/components/deployments/molecules/deploymentTableHeader.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { translate } from "@i18n";
+import { HeaderCell } from "@react-components/atoms/table";
+import { TableHeader } from "@react-components/molecules/table";
+
+export const DeploymentTableHeader: React.FC = () => {
+	return (
+		<TableHeader classes="sticky top-12 z-30">
+			<HeaderCell>{translate().t("reactApp.deployments.time")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.deployments.status")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.sessions.statuses.stopped")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.sessions.statuses.running")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.sessions.statuses.error")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.sessions.statuses.completed")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.deployments.buildId")}</HeaderCell>
+			<HeaderCell>{translate().t("reactApp.deployments.actions")}</HeaderCell>
+		</TableHeader>
+	);
+};
```

Comment:
What do u think this? And not need use React.FC
```suggestion
export const DeploymentTableHeader = () => {
    const t = translate().t;
    const keys = [
        "reactApp.deployments.time",
        "reactApp.deployments.status",
        "reactApp.sessions.statuses.stopped",
        "reactApp.sessions.statuses.running",
        "reactApp.sessions.statuses.error",
        "reactApp.sessions.statuses.completed",
        "reactApp.deployments.buildId",
        "reactApp.deployments.actions"
    ];

    return (
        <TableHeader classes="sticky top-12 z-30">
            {keys.map(key => (
                <HeaderCell key={key}>{t(key)}</HeaderCell>
            ))}
        </TableHeader>
    );
};
```

Reply:
We prefer to keep the code as simple as we can, so instead of `map` we prefer to use the `HeaderCell` one by one.